### ISSUE TITLE
Update to latest MongoDB

### ISSUE THREAD

**Issue by jessbowers** [cdimascio/uuid-mongodb#74]
```
warning " > uuid-mongodb@2.5.1" has incorrect peer dependency "mongodb@3.6.x || 3.7.x || 4.0.x || 4.1.x || 4.2.x || 4.3.x || 4.4.x || 4.5.x".
```

latest: 4.10.0

Is it still necessary to pin these dependencies? Does the mongodb project follow SemVer?

**Comment by jdegger** [comment]
Any updates here?

**Comment by cdimascio** [comment]
Available in v2.5.2
